lib/typingsInstaller.js

```diff
@@ -11555,27 +11555,31 @@ var ts;
             isBindableStaticNameExpression(expr.arguments[0], /*excludeThisKeyword*/ true);
     }
     ts.isBindableObjectDefinePropertyCall = isBindableObjectDefinePropertyCall;
-    function isBindableStaticElementAccessExpression(node, excludeThisKeyword) {
-        return isLiteralLikeElementAccess(node)
-            && ((!excludeThisKeyword && node.expression.kind === 103 /* ThisKeyword */) ||
-                isEntityNameExpression(node.expression) ||
-                isBindableStaticElementAccessExpression(node.expression, /*excludeThisKeyword*/ true));
-    }
-    ts.isBindableStaticElementAccessExpression = isBindableStaticElementAccessExpression;
+    /** x.y OR x[0] */
     function isLiteralLikeAccess(node) {
         return ts.isPropertyAccessExpression(node) || isLiteralLikeElementAccess(node);
     }
     ts.isLiteralLikeAccess = isLiteralLikeAccess;
+    /** x[0] OR x['a'] OR x[Symbol.y] */
```

Comment:
Note: It is kind of silly that this doesn't work for negative numbers, but nobody's asked for it so...